cmd/fitgen/internal/fitstringer/errors.go

```diff
@@ -0,0 +1,10 @@
+package fitstringer
```

Comment:
This is manually "vendored" code (https://pkg.go.dev/golang.org/x/tools/cmd/stringer) with a few small changes for the `fitgen`. We should do no changes here to keep the diff with upstream as small as possible.